GUI fixes

blakkd opened the pull request:
- translated some comments
- set en as default locale
- fixed gradio OCR mode dropdown menu
- fallback to default gradio theme

XJF2332 commented:
Thank you for your contribution! However, I would like to point out that zh-cn (Simplified Chinese) should be the default locale, as the majority of users for this project are Chinese. This can be observed from the language preferences in issues, not only within this repository but also in the original repository.

If you prefer not to make this change, please let me know, and I will consider adding a branch, perhaps named `en` or `English`, which will use English as the default language.

blakkd commented:
Ok, I just reverted the default locale to zh-CN. I actually shouldn't have touched it, I don't really know why I did that. I think what happened is I changed the locale using Config Manager.py and when commiting later I forgot to add it to my .gitignore and thought I made the change on purpose. It was late night, sorry... You are totally right, better let it set to Chinese for the current users, especially when you wrote Config Manager.py for the purpose!
